config/deploy/after.sh

```diff
@@ -43,6 +43,13 @@ if [[ $ENVIRONMENT =~ "production" ]]; then
 "revision": "${REVISION_TITLE}"
 }
 EOF
+else
+  tee deploy_revision_payload.json <<EOF >/dev/null
+{
+"deploy_all": true,
+"revision": "${REVISION_TITLE}"
+}
```

Comment:
🗒️ No need to update it in this PR, but something to keep in mind in the future.

I know you fished this from history, but using `tee` with `>/dev/null` basically nulls the benefits of `tee`.

`tee` is like `cat`, but it also prints to screen, if we don't want to print the data to the log, we can use `cat`, if we do want the data to be printed we can remove `>/dev/null`.


```suggestion
cat > deploy_revision_payload.json <<EOF
{
"deploy_all": true,
"revision": "${REVISION_TITLE}"
}
```

same applies to all usages of `tee` in this file.